packaging/wrappers/grafana-cli

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+# Wrapper for the grafana-cli binary
+# This file serves as a wrapper for the grafana-cli binary. It ensures we set the system-wide Grafana configuration
+# that was bundled with the package as we use the binary.
+
+DEFAULT=/etc/default/grafana
+
+GRAFANA_HOME=/usr/share/grafana
+CONF_DIR=/etc/grafana
+DATA_DIR=/var/lib/grafana
+PLUGINS_DIR=/var/lib/grafana/plugins
+LOG_DIR=/var/log/grafana
+
+CONF_FILE=$CONF_DIR/grafana.ini
+PROVISIONING_CFG_DIR=$CONF_DIR/provisioning
+
+EXECUTABLE=$GRAFANA_HOME/grafana-cli
+
+if [ ! -x $EXECUTABLE ]; then
+ echo "Program not installed or not executable"
+ exit 5
+fi
+
+if [ $# -eq 0 ]; then
+  eval $EXECUTABLE
+  exit 1
+fi
+
+# overwrite settings from default file
+if [ -f "$DEFAULT" ]; then
+  . "$DEFAULT"
+fi
+
+OPTS="--homepath=${GRAFANA_HOME} \
+      --config=${CONF_FILE} \
```

Comment:
So with this I assume we do not need to do the `if dev { change pluginsDir }` we have now in the CLI right? It should pick up the one in config? In the config though the setting is `plugins = data/plugins` for example so is this picked somehow? Also this will need to be appended to homepath but that is also different when doing development so not sure if that is handled somehow.

Reply:
> So with this I assume we do not need to do the if dev { change pluginsDir } we have now in the CLI right?

Sorry @aocenas 🙈, not sure I follow the questions. I'm not entirely sure where `if dev { change pluginsDir }` is coming from.

>  It should pick up the one in config? In the config though the setting is plugins = data/plugins for example so is this picked somehow? 

That is correct, because of:

```bash
# overwrite settings from default file
if [ -f "$DEFAULT" ]; then
  . "$DEFAULT"
fi
``` 

We load the settings from the default file and override with whatever we have here. 


> Also this will need to be appended to homepath but that is also different when doing development so not sure if that is handled somehow.

Not sure I follow that last question but if what we mean by development is us developing in our local machines this wouldn't affect it as the wrapper only affects RMP and deb packaging.

Reply:
Sorry this was probably a bit cryptic. In the CLI there is this part https://github.com/grafana/grafana/blob/be66ed9dabcc7d8d76510b39330d523e3efe1da2/pkg/cmd/grafana-cli/utils/grafana_path.go#L11 where we try to get the plugin dir when you are on dev environment and actually figure out if we are in dev environment. It does not work very well and the diff that @bergquist ping you on in my PR is my trying to fix that for second time.

So I think there could be some more robust solution like getting it from the config.ini where you already need need to setup it correctly so that grafana itself knows where to find installed plugins. At the same time I wanted it to work no matter what is the current cwd (like if I run ./bin/grafana-cli or ./grafana-cli or ./bin/darwin-amd64/grafana-cli ideally I would like the same results).  

> he wrapper only affects RMP and deb packaging.

In this case though this probably isn't going to help so we will need to fix the paths on dev in other PR anyway.

Reply:
Agreed! Sorry, but this wouldn't really help - I'm pretty sure we could think of something but it feels like it sits outside of the scope of this PR.